omero/developers/Web/CreateApp.rst

```diff
@@ -6,86 +6,93 @@ to get you familiar with the Django framework. The more you know about
+app is on the :envvar:`PYTHONPATH`, described in more detail below.
+We also choose the OMERO server we want to connect to.
+You need to edit the ``demo.openmicroscopy.org`` and ``/path/to/dir/`` in the
+following command:
 
-Your app needs to be within a directory that is on your :envvar:`PYTHONPATH`.
-We usually create a new container for a new app, and add it to the
-:envvar:`PYTHONPATH`.
 ::
 
-    $ mkdir organization-appname
-    $ cd organization-appname
-    $ export PYTHONPATH=$PYTHONPATH:/path/to/organization-appname
+    $ docker run -it -e OMEROHOST=demo.openmicroscopy.org -p 4080:4080 -e CONFIG_omero_web_apps='["minimal_webapp"]' -v /path/to/dir/omero-demo-webapp/minimal-webapp/minimal_webapp:/opt/omero/web/venv/lib/python2.7/site-packages/minimal_webapp openmicroscopy/omero-web-standalone
 
+This will run Docker interactively in the terminal.
+Now go to `http://localhost:4080/minimal_webapp/ <http://localhost:4080/minimal_webapp/>`_
+in your browser.
+You should be redirected to the login screen and then back to the ``minimal_webapp``
+page which will display your Name and list your Projects.
 
-OR you could simply choose an existing location:
+Adding your app to your PYTHONPATH
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-::
+Your Django app is the directory that contains the ``urls.py`` and ``views.py``. In the
+example above, this is the ``minimal_webapp`` directory.
+This needs to be *within* a directory that is on your :envvar:`PYTHONPATH` or
+`python search path <https://docs.python.org/2/install/index.html#modifying-python-s-search-path>`_.
 
-    $ cd /somewhere/on/your/pythonpath/
+In the Docker example above, we mount the ``minimal_webapp`` directory within the
+``site-packages`` of the Docker image.
 
+If you have installed OMERO.web locally in a virtual environment, you can add a
+path configuration file to the ``site-packages`` directory.
+This must specify the folder that *contains* ``minimal_webapp``
+(in this case it is the parent ``minimal-webapp`` directory) to
+be added to the ``sys.path``.
 
-Create and check out a new GitHub repository OR manually create a new directory
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+::
 
--  Login to your GitHub account homepage e.g.
-   `<https://github.com/>`_\<your-name>/) and click
-   "New repository"
+    # Find where your python is
+    $ which python
+    /absolute-path/to-your/Virtual/omero/bin/python
 
--  Enter the name of your repository e.g. ``<organization-appname>``,
-   add a ``README.rst`` file (using the ``.rst`` extension allows it to be rendered correctly on PyPI).
-   Its contents should cover the goal of the app and the configuration instructions.
+    # Create a path configuration file .pth in site-packages
+    $ echo /path/to/dir/omero-demo-webapp/minimal-webapp >> /absolute-path/to-your/Virtual/omero/lib/python2.7/site-packages/minimal_webapp.pth
```

Comment:
I do not suppose that the name of the venv under which the web is installed must be always "omero" - in that case, the path would be a bit different I suppose.

Reply:
yes - I'll fix